Source/WebKit/NetworkProcess/NetworkLoad.cpp

```diff
@@ -61,7 +61,7 @@ NetworkLoad::NetworkLoad(NetworkLoadClient& client, NetworkLoadParameters&& para
         m_task = NetworkDataTask::create(networkSession, *this, m_parameters);
 }
 
-NetworkLoad::NetworkLoad(NetworkLoadClient& client, NetworkSession& networkSession, const Function<RefPtr<NetworkDataTask>(NetworkDataTaskClient&)>& createTask)
+NetworkLoad::NetworkLoad(NetworkLoadClient& client, NetworkSession& networkSession, NOESCAPE const Function<RefPtr<NetworkDataTask>(NetworkDataTaskClient&)>& createTask)
```

Comment:
You don't need to specify NOESCAPE in the function definition. Just put that on declaration.